tools/PI/DevHome.PI/DevHome.PI.csproj

```diff
@@ -0,0 +1,214 @@
+    <EnableMsixTooling>false</EnableMsixTooling>
+    <Nullable>enable</Nullable>
+    <LanguageVersion>12.0</LanguageVersion>
+    <StartupObject>DevHome.PI.Program</StartupObject>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <EnableDefaultApplicationDefinition>false</EnableDefaultApplicationDefinition>
+    <DefineConstants>$(DefineConstants);DISABLE_XAML_GENERATED_MAIN</DefineConstants>
+
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <CsWinRTWindowsMetadata>10.0.19041.0</CsWinRTWindowsMetadata>
+    <PackageIcon>PI.ico</PackageIcon>
+    <ApplicationIcon>PI.ico</ApplicationIcon>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <None Remove="BarWindow.xaml" />
+    <None Remove="Controls\AppStatusBar.xaml" />
+    <None Remove="Controls\ExpandedViewControl.xaml" />
+    <None Remove="Controls\GlowButton.xaml" />
+    <None Remove="Controls\SystemBarHorizontal.xaml" />
+    <None Remove="Controls\SystemBarVertical.xaml" />
+    <None Remove="errors.db" />
+    <None Remove="Images\PI.ico" />
+    <None Remove="Pages\AppDetailsPage.xaml" />
+    <None Remove="Pages\EtwPage.xaml" />
+    <None Remove="Pages\EventsPage.xaml" />
+    <None Remove="Pages\HelpPage.xaml" />
+    <None Remove="Pages\InsightsPage.xaml" />
+    <None Remove="Pages\ModulesPage.xaml" />
+    <None Remove="Pages\ProcessListPage.xaml" />
+    <None Remove="Pages\ResourceUsagePage.xaml" />
+    <None Remove="Pages\WatsonsPage.xaml" />
+    <None Remove="Pages\WinLogsPage.xaml" />
+    <None Remove="SettingsUi\AddToolControl.xaml" />
+    <None Remove="SettingsUi\EditToolsControl.xaml" />
+    <None Remove="SettingsUi\SettingsToolWindow.xaml" />
+  </ItemGroup>
+
+  <!-- Needed to avoid conflicts with DevHome's App.xaml and to let
+       XAML know where the main application object is defined-->
+  <ItemGroup>
+    <Page Remove="PIApp.xaml" />
+    <ApplicationDefinition Include="PIApp.xaml">
+      <Generator>MSBuild:Compile</Generator>
+      <XamlRuntime>$(DefaultXamlRuntime)</XamlRuntime>
+    </ApplicationDefinition>
+  </ItemGroup>
+
+  <ItemGroup>
+    <Content Include="Assets\SplashScreen.scale-200.png" />
+    <Content Include="Assets\Square150x150Logo.scale-200.png" />
+    <Content Include="Assets\Square44x44Logo.scale-200.png" />
+    <Content Include="Assets\Wide310x150Logo.scale-200.png" />
+    <Content Include="errors.db">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+    <Content Include="Images\PI.ico">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+    <Content Include="PI.ico" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Data.Sqlite" Version="8.0.3" />
+    <PackageReference Include="Microsoft.Diagnostics.Tracing.TraceEvent" Version="3.1.9" GeneratePathProperty="true"/>
+    <PackageReference Include="Microsoft.Extensions.Configuration.CommandLine" Version="8.0.0" />
+    <PackageReference Include="Serilog" Version="3.1.1" />

```

Comment:
Since you ProjectReference DevHome.Common, you can remove a bunch of these PackageReferences (especially Serilog and WASDK, were more PackageReferences mean more places to keep in sync that have been problematic in the past)

Reply:
Good call, a few things still need to be referenced like cswin32 or else there are compilation errors.